patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -25,90 +25,6 @@ const utilNavNarrowCheck = function() {
   return utilNarrowNav ? (utilNarrowNav.offsetWidth > 0 && utilNarrowNav.offsetHeight > 0) : false;
 };
 
-/** DP-19336 begin: add padding to hamburger menu to allow scrolling when alerts are loaded */
```

Comment:
@ygannett Since we are setting fixed positioning now when we open the menu, we don't need this. The height of the alerts does not affect the scrollable space of the menu. I removed this because some iOS devices were crashing, and I wanted to eliminate anything that might be causing any excessive processing or memory usage.

Reply:
@tom-fleming Sounds good.